packages/cli/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@warthog/cli",
+  "version": "0.0.1-alpha.2",
+  "description": "Warthog CLI",
+  "main": "dist/cli.js",
+  "types": "dist/types/cli.d.ts",
+  "bin": {
+    "warthog": "bin/warthog"
+  },
+  "files": [
+    "bin",
+    "dist",
+    "LICENSE",
+    "README.md"
+  ],
+  "author": {
+    "name": "Dan Caddigan",
+    "url": "https://github.com/goldcaddy77"
+  },
+  "homepage": "https://github.com/goldcaddy77/warthog/tree/master/packages/cli#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/goldcaddy77/warthog.git"
+  },
+  "keywords": [
+    "graphql",
+    "typeorm",
+    "typescript"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/goldcaddy77/warthog/issues"
+  },
+  "dependencies": {
+    "cosmiconfig": "^5.2.1",
+    "gluegun": "^3.3.1",
+    "graphql": "^14.1.1",
+    "prettier": "^1.18.2",
+    "reflect-metadata": "^0.1.12",
+    "ts-node": "^7.0.1",
+    "typeorm": "^0.2.18",
+    "typeorm-typedi-extensions": "^0.2.3",
+    "typescript": "^3.7.2"
+  },
+  "devDependencies": {
```

Comment:
Have you tried this built? I image a bunch of these are real dependencies, not devDependencies

Reply:
Yes, I have tried the built modules - in both existing and new projects - and it is working fine. However, that is probably because `@warthog/core` have the CLI covered. I did a review of  dependencies that each module/sub-packages uses at 931326b, mostly based on `import` and `require` statements. Do you still see dependencies that should be real dependencies instead of devDependencies (or vice-versa)? Since you know the project better, it would be interesting if you could also reviewed it.